addons/mail/static/src/chatter/web/chatter.xml

```diff
@@ -10,18 +10,18 @@
                     'btn-secondary': state.composerType === 'note',
                     'active': state.composerType === 'message',
                     'my-2': !props.compactHeight
-                }" t-att-disabled="!state.thread.hasWriteAccess and !(state.thread.hasReadAccess and state.thread.canPostOnReadonly) and props.threadId" data-hotkey="m" t-on-click="() => this.toggleComposer('message')">
+                }" t-att-disabled="isReadonly" data-hotkey="m" t-on-click="() => this.toggleComposer('message')">
```

Comment:
These conditions are not correct, the true value depends on `_mail_post_access` for each type of document.

Reply:
`canPostOnReadonly` checks whether `_mail_post_access` is `read` or not.